README.md

```diff
@@ -16,64 +15,71 @@ Listen to the corresponding hourly music from the main Animal Crossing titles:
   - Wild World / City Folk
   - New Leaf
 
-Rain and snow variants are also included!
+'Raining' and 'snowing' variants are also included!
 
 ### Live Weather 🌧
 You can enable Weather Detection so weather variants of the hourly music can be automatically picked according to the weather in your location!
 
 ### K.K. Slider 🎙
-Listen to the mitic K.K. Slider songs! You can choose whether to keep his appearances to the usual Saturday nights, or to play his songs 24/7.
+Listen to the mitic K.K. Slider songs! The extension features both 'live' and 'aircheck' version of his tunes.
+
+You can choose whether to keep his appearances to the usual Saturday nights, or to play his songs 24/7.
+
 
 ### Town Tune 🎵
-You can customize your own Town Tune, which will be played every time an hour passes!
+The extension features an awesome Town Tune editor available within the browser.
 
+With it you can compone your custom Town Tune, which will be played every time an hour passes (just as in the game!)
 
 ## Installation
-The changes in this repository **remain unpublished** in any way. 
+Notice that this project remains unpublished right now. You'll soon be able to install it through the Chrome Web Store!
 
-Ideally, we would like to update the [original extension](https://chrome.google.com/webstore/detail/animal-crossing-music/fcedlaimpcfgpnfdgjbmmfibkklpioop) so project development could continue with the same user-base. Otherwise, this project will be published as a new extension.
+### Source Code
+You can [download](https://github.com/PikaDude/Animal-Crossing-Music-Extension/archive/develop.zip) the source code of the project and install the unpacked extension yourself.
```

Comment:
Even though `develop` was the base branch I suspect users were clicking this link and therefore downloading the broken `master`. Notice I changed it to download `develop`. We should revert this once we make `master` stable again.

Reply:
I have also added a download button near the top of the README 👍

Reply:
![chrome_2020-01-17_16-05-36](https://user-images.githubusercontent.com/18642415/72585977-1a4c7880-3944-11ea-99ff-20f78b35bc43.png)
I was kinda referring to the GitHub download button lol

Reply:
Oh I know but I'd expect that people either 1) are not familiar with Github and need instructions, therefore '**clone** or download may be tricky for them, or 2) already knows how Github works and wouldn't need instructions in the first place

Reply:
True true

Reply:
That's just my opinion though, do you think we should still remove the download badge and/or tell them to click the github button?

Reply:
Yeah otherwise it might get confusing